### ISSUE TITLE
[Backend] Praserer

### ISSUE THREAD

**Issue by kwk2696** [skkuse02/2019fall_41class_team4#10]
1. 한글을 Client로 전송할경우 깨져서 나오는 문제 해결. 
    Client에서 인코딩을 해줘야 하나?

**Comment by jaewan7599** [comment]
Yes. I think client send message by encoding, of course. But, Server should know the type of encoding function in client, so that server can decode it.
